.gitignore

```diff
@@ -1 +1,5 @@
 .*
+
+# Composer Dependencies
+/vendor
+/composer.lock
```

Comment:
I don't recommend excluding `composer.lock` from being committed because it ensures that dependencies for repo contributors will always be the same.